ga4gh/datamodel/__init__.py

```diff
@@ -187,14 +187,12 @@ def obfuscate(cls, idStr):
         fashion. This is not intended for security purposes, but rather to
         dissuade users from depending on our internal ID structures.
         """
-        return base64.b64encode(idStr)
+        return base64.urlsafe_b64encode(str(idStr)).replace(b'=', b'')
 
     @classmethod
-    def deobfuscate(cls, idStr):
-        """
-        Reverses the obfuscation done by the :meth:`obfuscate` method.
-        """
-        return base64.b64decode(idStr)
+    def deobfuscate(cls, data):
+        return base64.urlsafe_b64decode(str((
```

Comment:
An quick inline comment might be helpful here. This is going to look mysterious to someone coming in to the codebase fresh.